product_docs/docs/pgd/5/reference/pgd-settings.mdx

```diff
@@ -60,6 +60,15 @@ above.
 Setting the replica identity of tables to `full` increases the volume of
 WAL written and the amount of data replicated on the wire for the table.
 
+!!! warning On setting bdr.default_replica_identity to default
+When setting `bdr.default_replica_identity` to `default` using `ALTER SYSTEM`
+always quote the value, like this:
+```sql
+ALTER SYSTEM SET bdr.default_replica_identity="default";
+```
+This is because default, unquoted, is a special value to the [ALTER SYSTEM](https://www.postgresql.org/docs/current/sql-altersystem.html) command which triggers the removal of the setting from the configuration file. This results in the system using the actual default setting which is `auto`.
```

Comment:
```suggestion
This is because default, unquoted, is a special value to the [ALTER SYSTEM](https://www.postgresql.org/docs/current/sql-altersystem.html) command which triggers the removal of the setting from the configuration file. This results in the system using the PGD default setting which is `auto`.
```